doc/users/next_whats_new/3d_plot_pan_zoom.rst

```diff
@@ -0,0 +1,8 @@
+3D plot pan and zoom buttons
+----------------------------
+
+The pan and zoom buttons in the toolbar of 3D plots are now enabled.
+Deselect both to rotate the plot. When the zoom button is pressed,
```

Comment:
```suggestion
Unselect both to rotate the plot. When the zoom button is pressed,
```
or, extend the  previous sentence: ", if neither button is selected the plot can be freely rotated."